boards/shields/weact_ov2640_cam_module/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Charles Dias
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_WEACT_OV2640_CAM_MODULE
+
+config VIDEO_HFLIP
+	default n
+
+config VIDEO_VFLIP
+	default n
```

Comment:
I think this is where Twister complains now: it needs something like `bool Description of this`.

Reply:
```
warning: VIDEO_HFLIP (defined at boards/shields/weact_ov2640_cam_module/Kconfig.defconfig:6) defined without a type

warning: VIDEO_VFLIP (defined at boards/shields/weact_ov2640_cam_module/Kconfig.defconfig:9) defined without a type

error: Aborting due to Kconfig warnings
```